packages/daemon/src/app/controller/job.ts

```diff
@@ -142,14 +109,10 @@ export class JobController {
   @get('/:id')
   public async get() {
     const { id } = this.ctx.params;
-    try {
-      const job = await this.pipelineService.getJobById(id);
-      if (!job) {
-        throw new Error('job not found');
-      }
-      successRes(this.ctx, { data: job });
-    } catch (err) {
-      failRes(this.ctx, { message: err.message });
+    const job = await this.pipelineService.getJobById(id);
+    if (!job) {
+      throw new Error('job not found');
```

Comment:
```suggestion
      throw new TypeError('job not found');
```

Use TypeError as the default error.

Reply:
I'm using `ctx.throw()` instead.